spec/factories/events.rb

```diff
@@ -45,6 +45,24 @@
       deadline { Date.current - 1 }
     end
 
+    trait :with_teams do
+      transient do
+        teams_count 5
```

Comment:
Why do those values have to be fixed here? Can't we specify them randomly?

Reply:
teams_count is just a default value here. You can override it by setting `teams_count: your number`